usr/share/rear/prep/COVE/default/400_prep_cove.sh

```diff
@@ -0,0 +1,20 @@
+#
+# Prepare stuff for Cove
+#
+
+REQUIRED_PROGS+=( "${REQUIRED_PROGS_COVE[@]}" )
+
+if has_binary curl ; then
+    REQUIRED_PROGS+=( curl )
+else
+    REQUIRED_PROGS+=( wget )
```

Comment:
Please add one of them to your `REQUIRED_PROGS`. We already have `curl` in `PROGS` so I would recommend that one.